Src/beebwin.cpp

```diff
@@ -1609,11 +1821,17 @@ LRESULT CALLBACK WndProc(HWND hWnd,     // window handle
 
 		case MM_JOY1MOVE:
 			mainWin->ScaleJoystick(LOWORD(lParam), HIWORD(lParam));
+			mainWin->TranslateJoystick(0);
 			break;
 
 		case MM_JOY1BUTTONDOWN:
 		case MM_JOY1BUTTONUP:
```

Comment:
Do we also need to handle `MM_JOY2BUTTONDOWN` and `MM_JOY2BUTTONUP`?

Reply:
Pushing or releasing joystick button generates `MM_JOY2MOVE` in addition to `MM_JOY2BUTTONDOWN` or `MM_JOY2BUTTONUP`. There's no need to handle those events separately.